database/reindex_elastic.js

```diff
@@ -83,18 +91,21 @@ const prepareIndex = async () => {
   process.stdout.write(`Creating index ${config.defaultTenant.indexName}...\r\n`);
   process.stdout.write(' - Base properties mapping\r\n');
 
-  try {
-    await request.put(getIndexUrl(), elasticMapping);
-  } catch (_e) {
-    setTimeout(async () => {
-      await request.put(getIndexUrl(), elasticMapping);
-    }, 1000);
-  }
+  await fetch(getIndexUrl(), {
+    headers,
+    method: 'PUT',
+    body: JSON.stringify(elasticMapping),
+  });
+
```

Comment:
I was wondering about this timeout and agree we probably don't need it and it was already a smell of something wrong with the process flow and library resets.  Nice!